tasks.py

```diff
@@ -519,3 +519,11 @@ def checkpatch(ctx):
     except UnexpectedExit:
         # Will exit non-zero if no double-space-after-period lines are found.
         pass
+
+
+@task
+def linter(ctx):
```

Comment:
Let's call this `lint` to be consistent with the rest of the tasks which are in the imperative tense.